mathesar_ui/src/components/select/Select.svelte

```diff
@@ -37,10 +47,9 @@
   </svelte:fragment>
   
   <svelte:fragment slot="content">
-    <ul>
+    <ul id="select-value-{selectId}" tabindex="0" role="listbox" aria-expanded="true">
```

Comment:
The aria-controls within Dropdown component should be same as this id.

Please pass this id and the label to the Dropdown component.
```suggestion
   export let ariaLabel;

   <Dropdown ariaControls="select-value-{selectId}" {ariaLabel} ...>
   ...
    <ul id="select-value-{selectId}" tabindex="0" role="listbox" aria-expanded="true">
```

And within the Dropdown component, ariaLabel seems to be handled. Please handle:

```suggestion
export let ariaControls;

<Button ... aria-controls={ariaControls} ...>
```